README.md

```diff
@@ -64,9 +64,21 @@ Then open the following URL <http://0.0.0.0:5000/>.
 ![image](dashboard_preview_map.png)
 
 
-## Running Tests
+## Testing
 
-Within your python environment run the following command to run the tests:
+### Test Requirements
+The testing suite requires [Dash Testing](https://dash.plotly.com/testing) and [pytest-mock](https://pypi.org/project/pytest-mock/), which can be installed in your python environment by running:
 ```
-python -m unittest
+python -m pip install dash\[testing]
+```
+then
+```
+pip install pytest-mock
+```
```

Comment:
Why are we using two different approaches with `pip` to install requirements?
`python -m pip install ...` vs `pip install..`.
I think the `python -m pip ...` will work around a messed up python/pip installation.
Also why not just install both requirements with a single command?

Reply:
I copied the installation instructions from [Dash](https://dash.plotly.com/testing). If it's not an issue to install both together with just `pip install ...`, then there's no reason not to.